Docs/Articles/Containerizing.md

```diff
@@ -0,0 +1,29 @@
+# Containerizing & Kubernetesizing Apps
```

Comment:
Please rename Docs to docs.
Please rename this header to `Notes on Containerizing Apps`
Please move all of this content to the bottom of the main Readme and delete this page.